custom/enikshay/integrations/nikshay/repeater_generator.py

```diff
@@ -405,21 +405,11 @@ def _get_person_case(self, episode_case):
         return get_person_case_from_episode(episode_case.domain, episode_case.get_id)
 
     @staticmethod
-    def _get_treatment_initiation_status(episode_case_properties):
-        """
-        :return: mapped value for dst status if available else fallback to
-        treatment_initiation_status if available else
-        finally send NOT_AVAILABLE_VALUE
-        """
-        dst_status = episode_case_properties.get('dst_status')
-        if dst_status:
-            if dst_status == 'pending':
-                return NOT_AVAILABLE_VALUE
-            if dst_status == 'rif_resistant':
-                return drug_susceptibility_test_status.get('rif_resistant')
-            if dst_status == 'rif_sensitive':
-                return drug_susceptibility_test_status.get('rif_sensitive')
-        return episode_case_properties.get('treatment_initiation_status', NOT_AVAILABLE_VALUE)
+    def _get_fh_name(person_case_properties):
```

Comment:
nit: is `fh` supposed to stand for `husband_father` if so might be nicer to make it `hf` for consistency. or just `_get_husband_father_name`